releasenotes/notes/docker-event-filtering-fix-2d952f26f8cfa76f.yaml

```diff
@@ -0,0 +1,5 @@
+---
+features:
+  - Docker check: ignore the new exec_die event type by default
```

Comment:
only the fix line is enough no?

Reply:
There's also a change in default behaviour (adding a new exclusion), that I'll backport to A5.
If a client has a custom exclusion list, they need to add it to theirs (as the defaults are only used if no list is given)